BraveWallet/Extensions/KeyringServiceExtensions.swift

```diff
@@ -0,0 +1,34 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import BraveCore
+
+extension BraveWalletKeyringService {
+  
+  // Fetches all keyrings for all given coin types
+  func keyrings(
+    for coins: [BraveWallet.CoinType]
```

Comment:
This could probably be `Set<BraveWallet.CoinType>` instead

Reply:
fixed in [1aaf290](https://github.com/brave/brave-ios/pull/5746/commits/1aaf290ebb80a10973b88d03bca9c0ed1647cf34)